ConsentViewController/Classes/Views/tvOS/NativePrivacyManager/GDPR/GDPRPMConsentSnapshot.swift

```diff
@@ -64,7 +64,7 @@ class GDPRPMConsentSnaptshot: NSObject, ConsentSnapshot, PMVendorManager, PMCate
         if disclosureCategories.isNotEmpty() {
             for category in disclosureCategories {
                 self.specialPurposes.insert(
-                    GDPRCategory(name: category.name, description: category.description, disclosureOnly: true, legIntVendors: category.disclosureOnlyVendors)
+                    GDPRCategory(name: category.name, description: category.description, disclosureOnly: true, legIntVendors: category.disclosureOnlyVendors, requiringConsentVendors: category.disclosureOnlyVendors)
```

Comment:
@Nevazhnovu is that correct?
```swift
 requiringConsentVendors: category.disclosureOnlyVendors
 ```

Reply:
@andresilveirah yes

Reply:
Disclosure Only Vendors exist in both tabs in the web solution so the above is correct

Reply:
@Nevazhnovu but as the name suggests, Disclosure Only Vendors, are vendors that don't require consent. They can't be toggled on or off.

Reply:
@andresilveirah Disclosure Only Vendors don't have any buttons.